e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/bss-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,80 @@
+import relative from '../../../relativeURL';
+import facilityPage from '../../../pages/facilityPage';
+import amendmentsPage from '../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../fixtures/deal-AIN';
+import dateConstants from '../../../../../../e2e-fixtures/dateConstants';
+import { PIM_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../../e2e-fixtures';
+
+context('Amendments - BSS/EWCS deal does not display any Facility end date pages or fields', () => {
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+      cy.createFacilities(dealId, [mockFacilities[0]], BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType, PIM_USER_1);
+    });
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, BANK1_MAKER1);
+    });
+  });
+
+  it('should not show any facility end date fields in the Details tab', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
+    facilityPage.facilityIsUsingFacilityEndDate().should('not.exist');
+    facilityPage.facilityFacilityEndDate().should('not.exist');
+    facilityPage.facilityBankReviewDate().should('not.exist');
+  });
+
+  it('should go straight from the cover end date amendments page to the review amendment answers page without displaying any facility end date fields', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
+
+    facilityPage.facilityTabAmendments().click();
+    amendmentsPage.addAmendmentButton().should('exist');
+    amendmentsPage.addAmendmentButton().contains('Add an amendment request');
+    amendmentsPage.addAmendmentButton().click();
+
+    cy.url().should('contain', 'request-date');
```

Comment:
could these be split up into different it statements?

Reply:
I don't think it's needed, I prefer longer E2E tests that check full journeys without having to restart the journey each time and increase the test length. I don't think there's a particularly clear or nice way to break it up.